website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/stylus-sdk.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Stylus SDK レポジトリ'
+description: A table with links and minimal information about the different SDKs
+author: rachel-bousfield
+translator: Ying@Fracton, Naoki@Fracton, Yu@Fracton
+sme: rachel-bousfield
+target_audience: Developers writing and deploying Stylus programs
+sidebar_label: 'Stylus SDK レポジトリ'
+sidebar_position: 2
+---
+
+import TranslationBannerPartial from "./partials/_stylus-translation-banner-partial.md";
```

Comment:
If you want to import from `reference` dir, you should change the path to `../partials/xxx`. (Not only this file, but all files under `reference`)
```suggestion
import TranslationBannerPartial from "../partials/_stylus-translation-banner-partial.md";
```